CDCgov/prime-reportstream
# 7139 - prevent org names with special characters/reserved words

Fixes #7139 This PR ...

Prevents org names with special characters/reserved words

Test Steps:
1. Log into RS
2. Edit an Organization
3. Select 'New' under 'Organization Sender Settings' or 'Organization Receiver Settings'
4. Try to enter a non-ascii character or a prohibited word and click 'Save'
5. You should not be able to save.
6. Click 'Cancel'
7. 'Clone' an existing setting
8. Try to enter a non-ascii character or a reserved word and click 'Save'
9. You should not be able to save.
*prohibited words - 
        "sender",
        "receiver",
        "org",
        "organization",
        "list",
        "all", 
        "revs"

## Changes
- Added logic when Saving
![Screenshot 2022-12-20 at 2 12 46 PM](https://user-images.githubusercontent.com/102491809/208776841-3f68a2bb-3790-4469-af3f-6155476b8a6f.png)


## Checklist

### Testing
